simulate/debug.gpd

```diff
@@ -0,0 +1 @@
+Test	0	+	0	14	2	14	3	2,6,11,	6,11,14,	0,2,0,
```

Comment:
Not the biggest deal but I was wondering if you got the file extension right? I usually see it as `file.gp`, but I am not sure.